CONTRIBUTING.md

```diff
@@ -73,6 +73,15 @@ The documentation is located in the `website` directory and can be started with
 
 There are other available commands too. As set up in the [.build](./.build/) directory.
 
+## How to Check the docs
```

Comment:
I think these should be included into the website tasks. @rstaib what do you think? should they be added to the docs build steps?